### PR TITLE
Update campaigns.en.md

### PR DESCRIPTION
**Pull request by realjenny**

### CODE REVIEW

**Review by butterflyhug — APPROVED** [review]

File: frontend/content/power/campaigns.en.md [hunk 1]
```diff
@@ -26,44 +26,10 @@ Sign and share the [letter to show your support for these educators](https://jus
 
 Google operates four Childcare Centers in the Bay Area that help provide subsidized childcare to Google workers by Google workers. These Google Childcare Center workers are being required to return to work while Google’s shuttle service transportation, which many rely on, remains suspended. When workers raised this issue, the corporate response was “transportation is just a perk, not a benefit.” Shifting this cost to essential workers, who earn far less than the Googlers whose children they care for, is unacceptable. Google must do better.
 
-## Data center workers: know our rights
```
Comment (junefish) [inline frontend/content/power/campaigns.en.md:29]: Should it be capitalised "Know Our Rights"?

**Review by junefish — CHANGES_REQUESTED** [review]
see comment in-line

File: frontend/content/power/campaigns.en.md (hunk 1 above)
Comment (junefish) [reply frontend/content/power/campaigns.en.md:29]: nvm

**Comment by junefish** [comment]
never mind

**Review by junefish — APPROVED** [review]

**Pull request merged**